pdns/webserver.hh

```diff
@@ -126,7 +126,12 @@ public:
   ComboAddress d_local;
 
   Socket *accept() {
-    return d_server_socket.accept();
+    Socket* socket = nullptr;
+    // handle EAGAIN
+    while (socket == nullptr) {
+      socket = d_server_socket.accept();
+    }
+    return socket;
```

Comment:
I'm sure I'm missing something but if the socket is in blocking mode we should never get `EAGAIN`, and if it's in non-blocking mode this looks like we would then enter busy-looping?